### PR TITLE
Fix for fetching gitlab subgroups and project branches

### PR DESCRIPTION
**Pull request by srikarm**

### CODE REVIEW

File: back-end/hub-api/src/test/java/io/apicurio/hub/api/gitlab/GitLabSourceConnectorTest.java [hunk 1]
```diff
@@ -217,7 +217,7 @@ public void testGetProjects() throws GitLabException, SourceConnectorException {
     @Test
     @Ignore
     public void testGetBranches() throws GitLabException, SourceConnectorException {
-        Collection<SourceCodeBranch> branches = service.getBranches("Apicurio", "api-samples");
+        Collection<SourceCodeBranch> branches = service.getBranches( "api-samples");
```
Comment (EricWittmann) [inline back-end/hub-api/src/test/java/io/apicurio/hub/api/gitlab/GitLabSourceConnectorTest.java:220]: I'm assuming I will need to lookup the unique projectId for the `Apicurio/api-samples` project on GitLab, yes?

Comment (EricWittmann) [reply back-end/hub-api/src/test/java/io/apicurio/hub/api/gitlab/GitLabSourceConnectorTest.java:220]: Otherwise this will not work?

**Comment by EricWittmann** [comment]
I'm just waiting on CircleCI to validate/invalidate my changes to the PR. :)

File: back-end/hub-api/src/test/java/io/apicurio/hub/api/gitlab/GitLabSourceConnectorTest.java (hunk 1 above)
Comment (srikarm) [reply back-end/hub-api/src/test/java/io/apicurio/hub/api/gitlab/GitLabSourceConnectorTest.java:220]: Yes. That would be correct.

**Comment by EricWittmann** [comment]
Thanks for the contribution!

**Pull request merged**